views/movies.hbs

```diff
@@ -0,0 +1,14 @@
+<h2>Ironhack Cinema</h2>
+<p>Click on the movies to check the showtimes!</p>
+
+<div class='movies-list'>
+{{#each movies}}
+<div class='movies-details'>
+ 
+ <img src="{{this.image}}" alt="{{this.title}} img">  
+ {{this.title}}
+<a href="/see-more{{id}}">See more</a>     
```

Comment:
I'm not sure you have a route that handles this --> `/see-more{{id}}` in your index.js. 
Maybe you mean to use
```suggestion
<a href="/movies/:{{id}}">See more</a>     
```